sklearn/metrics/_scorer.py

```diff
@@ -652,6 +653,8 @@ def make_scorer(score_func, *, greater_is_better=True, needs_proba=False,
 neg_mean_gamma_deviance_scorer = make_scorer(
     mean_gamma_deviance, greater_is_better=False
 )
+neg_pinball_loss_scorer = make_scorer(pinball_loss,
+                                      greater_is_better=False)
```

Comment:
Please register this in the dict of scores named `SCORERS` below in the same file.